src/app/recipes/recipes.service.ts

```diff
@@ -94,7 +93,7 @@ export class RecipesService {
   ];
 
   private emptyRecipe: Recipe = {
-    title: 'nothing here',
+    title: 'title',
```

Comment:
store value like this in separate .constants file, dont pollute classes with object like this